pipelines/coref/project.yml

```diff
@@ -0,0 +1,117 @@
+title: "Training a spaCy Coref Model"
+description: >
+  This project trains a coreference model for spaCy using OntoNotes.
+
+vars:
+  # XXX Change to your actual GPU ID
+  gpu_id: 0
+  # XXX fill with your local path to OntoNotes
+  ontonotes: "/home/USER/ontonotes5/data"
+  max_epochs: 20
```

Comment:
Is it reasonable to assume that this will be the same value for `train_cluster` and `train-span-predictor`?

Reply:
Not sure exactly which one you're referring to, but yes for the gpu and ontonotes path. 

For the epochs, currently it makes sense to keep them the same. In practice the span predictor trains much more quickly, so the limit could be lower there, but because of the patience setting it doesn't really matter that the max epochs are high.

Reply:
Ah yes I meant the `max_epochs` only, I guess Github just shows a few previous lines by default. Ok thanks!